Basic Update to avoid conflict in the future

GuillaumeDespaux opened the pull request.

GuillaumeDespaux committed:
RACI implementation

GuillaumeDespaux committed:
KPI's implementation

First implementation of the kpi's
Need to details or rework a little bit the content for design and game section in it.
Also after that to full check the implementation the Team mood.

GuillaumeDespaux committed:
Weekly report W2 start

GuillaumeDespaux committed:
Merge branch 'main' into management

GuillaumeDespaux committed:
Create README.MD

GuillaumeDespaux committed:
Update weekly_report_2.md

The files has been completed for this week

GuillaumeDespaux committed:
KPI's clarification

Clarification about how the subpoints are defined

Enonome reviewed (changes requested):
- The $$ doesn't work in the ### Total Completion Calculation (lines 24, 26, 29 and 31)
- I'm not sure that "Gaant Chart" exist, maybe you wanted to say "Gantt Chart".
- In the 4. of the weekly report, 2nd point, there are "the" and "a"

Enonome commented:
Also, the "Main Points" mentionned are not explicits (I don't understand 😅). Maybe you can try naming it explicitly

Enonome reviewed (approved):
All is good for me

GuillaumeDespaux committed:
Update to QA asking modification

The following points are the requested point from the QA.

https://github.com/algosup/2024-2025-project-1-fpga-team-2/pull/5#pullrequestreview-2351419652

https://github.com/algosup/2024-2025-project-1-fpga-team-2/pull/5#issuecomment-2396420832

GuillaumeDespaux committed:
Double ligne mistake
